modules/pipeline/providers/dialeragent/interface.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dialeragent
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/rancher/remotedialer"
+
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/pkg/discover"
+)
+
+type Interface interface {
+	RegisterEdgeToDialer(ctx context.Context)
+}
+
+// RegisterEdgeToDialer only registers the edge to the dialer
+func (p *provider) RegisterEdgeToDialer(ctx context.Context) {
+	if !p.Cfg.IsEdge {
+		p.Log.Warnf("current node is not edge node, skip dialer agent registration")
```

Comment:
current Pipeline is not deployed on edge side, skip register to center dialer

Reply:
adjusted